project.clj

```diff
@@ -27,122 +32,94 @@
   :jvm-opts ["-Djava.awt.headless=true"
              "-Dlog.console.threshold=INFO"
              "-server"]
-  :pedantic? :warn
-  :dependencies [[org.clojure/clojure "1.9.0"]
-                 [clj-time "0.15.1"]
-                 [org.clojure/core.async "0.3.465" :exclusions [org.clojure/tools.reader]]
+  ; use `lein pom; mvn dependency:tree -Dverbose -Dexcludes=org.clojure:clojure`
+  ; to inspect conflicts.
+  :dependencies [[org.clojure/clojure ~clj-version]
+                 [clj-time "0.15.2"]
+                 [org.clojure/core.async "0.7.559"]
                  [org.slf4j/slf4j-log4j12 "1.8.0-beta0"]
-                 [org.clojure/core.memoize "0.7.2"]
+                 [org.clojure/core.memoize "0.8.2"]
                  [org.clojure/tools.logging "0.5.0"]
-                 [org.clojure/tools.cli "0.4.1"]
+                 [org.clojure/tools.cli "0.4.2"]
                  [pandect "0.6.1"]
 
                  ;; Schemas
-                 [prismatic/schema ~schema-version]
-                 [metosin/schema-tools ~schema-tools-version]
-                 [threatgrid/flanders "0.1.17"
-                  :exclusions [prismatic/plumbing
-                               potemkin
-                               com.andrewmcveigh/cljs-time]]
-                 [threatgrid/ctim "1.0.15"
-                  :exclusions [threatgrid/flanders
-                               metosin/ring-swagger
-                               com.google.guava/guava
-                               org.clojure/tools.reader
-                               org.clojure/clojurescript]]
-                 [threatgrid/clj-momo "0.3.3"]
+                 [prismatic/schema "1.1.12"]
+                 [metosin/schema-tools "0.12.2"]
+                 [threatgrid/flanders "0.1.23-20200204.180031-3"] ;sha: f92c65bb811e97cc0b96443e46474ba8caa1e1cc
+
+                  
+                 [threatgrid/ctim "1.0.16-20200205.182529-5"] ;sha: 121b661f9638d6720c2ac22f2bdd125383abdc8a
+                 [threatgrid/clj-momo "0.3.4-20200204.172456-2"] ;sha: 296203d2bc07e0063103e4bc1cc0c921284db927
+
```

Comment:
Don't forget to use the released version of `flanders`, `ctim` and `cli-momo` before merging

Reply:
Bumped flanders, ctim, clj-momo.